src/main/functions/DIDResolution.ts

```diff
@@ -31,6 +31,42 @@ export function getResolver(opts: ResolveOpts): Resolvable {
   }
 }
 
+export function getResolverUnion(subjectSyntaxTypesSupported: string[] | string, resolverMap: Map<string, Resolvable>): Resolvable {
+  const uniResolvers: {
+    [p: string]: (did: string, _parsed: ParsedDID, _didResolver: Resolver, _options: DIDResolutionOptions) => Promise<DIDResolutionResult>;
+  }[] = [];
+  const subjectTypes: string[] = [];
+  if (subjectSyntaxTypesSupported) {
+    typeof subjectSyntaxTypesSupported === 'string'
+      ? subjectTypes.push(subjectSyntaxTypesSupported)
+      : subjectTypes.push(...subjectSyntaxTypesSupported);
+  }
+  if (subjectTypes.indexOf(SubjectSyntaxTypesSupportedValues.DID.valueOf()) !== -1) {
+    return new UniResolver();
```

Comment:
This should be the customResolver if it is set btw, otherwise the UnionResolver.

So the first lines in getResolver which always returns the customResolver if set, is wrong. 

We need to populate the map values first. In case a resolver is set for the DID method, we will use that. In case a customResolver is set and the value is null we will use the customResolver for that method. In case no customResolver is set and the value is null, we will use the UniResolver

Reply:
changed to return the `customResolver` in the first line, the rest is changed according to our slack discussion